server/views/applications/pages/oasys-import/_oasys-screen.njk

```diff
@@ -1,9 +1,12 @@
 {%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
 {% from "../../../components/riskWidgets/macro.njk" import widgets %}
-
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% extends "../layout.njk" %}
 
 {% set columnClasses = "govuk-grid-column-full" %}
+{% set InsetText = "This information has been imported from OASys. You must:
```

Comment:
I think this would be easier to read if each sentence had its own line, EG: 
`<p>This information has been imported from OASys. You must:<br/>
Remove any out of date information or information that is not relevant to this application<br/>
Add in any other relevant information (please note - this does not write back to OASys).<br/>
Your application may be rejected if you do not do this.`

Reply:
I'd probably go one further and make it a bulleted list. You could set inside a block to make it more readable too:

```
{% set insetText %}
<p>This information has been imported from OASys. You must:</p>
<ul>
<li>Remove any out of date information or information that is not relevant to this application</li>
etc etc
</ul>
{% endset -%}
```

Reply:
Good shout

Reply:
{{ govukInsetText({
  text: .""
}) }} will only accept string not html

Reply:
ok got it

Reply:
@richpjames @pezholio please check again pushed changes